docs/walkthrough/isolates-intro.md

```diff
@@ -63,6 +64,7 @@ Isolate is just a fancy term we use when we run a serverless containers. Direkti
 functions:
 - id: httprequest
   image: vorteil/request:v2
+  type: reusable
 ```
 
 To use an Isolate it must first be defined at the top of the workflow definition. Each function definition needs an identifier that must be unique within the workflow definition, and an `image` that references a Docker container to use.
```

Comment:
Now that we have five types of functions, one of which we call "isolate", I don't think we can mix terminology. This article was written assuming "functions" and "isolates" are interchangeable. They're not any more. We should go through all the docs and change "isolate" to "function" in most places where it appeared before this update.